src/store/actions/search.ts

```diff
@@ -57,7 +58,14 @@ export const search = createAsyncThunk<
 					sortBy,
 					query: query || queryPart.join(' '),
 					offset,
-					types
+					types,
+					...(locale
+						? {
+								locale: {
+									_content: locale
+								}
+						  }
+						: undefined)
```

Comment:
in case there's no locale you will be spreading undefined, which will break the code. replacing undefined with {} will work